src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -207,13 +211,17 @@ export async function $onEmit(context: EmitContext<NetEmitterOptions>) {
 }
 
 function deleteFile(filePath: string) {
-    fs.unlink(filePath, (err) => {
-        if (err) {
-            console.log(`stderr: ${err}`);
-        }
-
-        console.log(`File ${filePath} is deleted.`);
-    });
+    if (fs.existsSync(filePath)) {
+        console.log(`File ${filePath} does not exist. No deletion. `);
```

Comment:
I don't think we should print this message.

Reply:
We need to check if the file exists or not, then unlink it, otherwise unlink will give out an error.

Reply:
We can check, but we don't have to `console.log`. It is strange to tell users that an "internal file is not exist, so we don't delete".